Makefile

```diff
@@ -528,6 +528,10 @@ endif
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=pca10059            examples/blinky2
 	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=bluemicro840        examples/blinky1
+	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=bluemicro840        examples/blinky2
+	@$(MD5SUM) test.hex
```

Comment:
One example should be good enough - feel free to pick any of the two.
(Each new board adds a second or so to CI time, and we have a lot of boards and a lot of PRs so I like to keep them to a minimum).